content/de/blog/hacktoberfest-2021/index.md

```diff
@@ -0,0 +1,125 @@
+{
+    "title": "Hacktoberfest 2021: Macht mit!",
+    "type": "blog",
+    "date": "2021-09-19T11:00:00+02:00",
+    "description": "Wir halten 2021 auch wieder eine Hacktoberfest-Aktion ab. Mach' bei Datenanfragen.de mit und gewinne eine Webcam-Abdeckung, Sticker und vielleicht ein T-Shirt!",
+    "featured_image": "hacktoberfest-2021",
+    "tags": [ "hacktoberfest", "digitalocean", "github", "open source", "pull request"],
+    "authors": [ "malte", "baltpeter" ],
+    "notices": "Titelfoto angepasst nach: „[maple leaves photo](https://unsplash.com/photos/Pu-en3ew8wY)“ von [Fey Marin](https://unsplash.com/@feymarin) ([Unsplash-Lizenz](https://unsplash.com/license))"
+}
+
+
+~~O'zapft is!~~ Der Oktober und damit auch das Hacktoberfest stehen wieder vor der Tür. Auch dieses Jahr wollen wir als Datenanfragen.de e.&thinsp;V. wieder am Hacktoberfest teilnehmen und freuen uns auf Eure Einreichungen für unser Projekt! Wie auch schon in den letzten Jahres gibt es auch wieder etwas zu gewinnen.
+
+{{< featuredImg alt="Vor einem Hintergrund aus Blättern steht: „Hacktoberfest 2021: Macht mit!“" >}}
+
+Wie auch das [ursprüngliche Hacktoberfest](https://hacktoberfest.digitalocean.com/) von DigitalOcean, wollen wir mit unserer Aktion Menschen die Mithilfe an Open-Source-Projekten näher bringen und freuen uns über jeden Beitrag für Datenanfragen.de oder eine unserer anderssprachigen Seiten. Konkret heißt das für Dich: Wenn Du an unserer Aktion teilnimmst und etwas zu Datenanfragen.de beiträgst, gewinnst Du eine praktische Webcam-Abdeckung und ein Sticker-Set (jeweils begrenzt auf die ersten 100 Teilnehmer_innen) und erhältst vielleicht auch ein T-Shirt von uns.
+
+Unser Projekt und unser Ziel Datenschutz möglichst allen zugänglich zu machen, leben von reger Beteiligung aus der Community. Besonders freuen wir uns über Beiträge in Gebieten, die wir ohne externe Hilfe nicht abdecken können: Übersetzungen von Deutsch und Englisch in andere in der EU gesprochene Sprachen, siehe z.&nbsp;B. [diesen Eintrag](https://github.com/datenanfragen/data/issues/229) in unserem Issue-Tracker. Oder weißt Du von [Firmen, die in bestimmten EU-Ländern Daten von allen Bürger_innen sammlen]?(https://github.com/datenanfragen/data/issues/230) Wenn Du uns diese mitteilst, können wir unseren Nutzer_innen aus diesen Ländern direkt diese Firmen als Anfrageempfängerinnen vorschlagen.
```

Comment:
I don't think we should have the "Besonders freuen wir uns…" sentence in the German version as per the previous discussion. Feel free to either remove it or replace it with some other good issues.

Reply:
I disagree, but sure. I want this merged.

Reply:
To elaborate on my reasoning: You are mentioning right in the introduction that we are mostly interested in contributions that most people reading this won't be able to help with. I'm afraid this will discourage them from reading further and participating at all as they may think our event isn't suited for them.

Reply:
Good point.